Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:stretch
```

Comment:
Right now the current version (aka "tag") of the Docker image (0.1.0) isn't stored anywhere (just where used in `.travis.yml`). If someone makes a change to the Dockerfile, they'd have to figure out how to build it, what tag to use, and where to put it.   Please add a `Makefile` for that.

Put the `Dockerfile` in a directory `.docker/` and the `Makefile` in that same directory.

Here's an example `Makefile` from one of our other projects: https://github.com/xaptum/xroute/blob/master/.docker/Makefile Don't worry about the separate `build.dockerfile` and `publish.dockerfile` in that repo.

Reply:
I have added a pretty rough attempt to add a `Makefile` for Docker. Right now it will try to pull the image from DockerHub on `make`, and if that fails it will check if that image exists, pushing the image if it does exist. If the image does not exist, and we couldn't pull it from DockerHub, `make build` will build the docker image.